ReadMe.md

```diff
@@ -51,10 +53,31 @@ setTimeoutAt(function (msg) {
 }, Date.now() + 5000, 'Hello world');
 ```
 
+To cancel a `Timeout`, use the instance's `clear` method, or pass it as an argument to `clearTimeout`:
+
+```js
+const st = require('safe-timers');
+const setTimeout = st.setTimeout;
+const clearTimeout = st.clearTimeout;
+
+let timeout1 = setTimeout(function (msg) {
+  console.log(msg);
+}, 5000, 'Hello world');
+let timeout2 = setTimeout(function (msg) {
+  console.log(msg);
+}, 5000, 'Hello world');
+
+timeout1.clear(); // this clears the timeout
+clearTimeout(timeout2); // this also clears the timeout!
+```
+
+### Interval
+
+To create an `Interval`, use the `setinterval` function:
+
 **Interval setInterval(Function callback, number interval, mixed arg1, mixed arg2, ...)**
 
-Calls `callback` after at least every `interval` milliseconds. All arguments passed after the `interval` will be passed
-to the callback when it gets invoked. Returns an `Interval` instance.
+- Creates and returns an `Interval` instance that will call `callback` after at least every `interval` milliseconds. All arguments passed after the `interval` will be passed to the callback when it gets invoked.
```

Comment:
I would recommend losing the `- ` characters at the start of this line as they are causing bulleted list formatting for the description of the method. The formatting looks odd and the indentation doesn't really line up well with the code examples and sometimes present subsequent non-bulleted supplemental sentences (e.g. about how to clear `Timeout`/`Interval` objects).